src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -512,39 +509,51 @@ private static bool AreEqualNetworkCredentials(NetworkCredential credential1, Ne
                 string.Equals(credential1.Password, credential2.Password, StringComparison.Ordinal);
         }
 
-        [Conditional(VerboseDebuggingConditional)]
-        private static void VerboseTrace(string text = null, [CallerMemberName] string memberName = null, EasyRequest easy = null, MultiAgent agent = null)
+        private static bool EventSourceTracingEnabled { get { return HttpEventSource.Log.IsEnabled(); } }
```

Comment:
> If "IsEnabled()" is a fast call, why isn't it a property?

Because there are overloads of it.

> Should the result of "IsEnabled()" be cached?

Tracing can be enabled and disabled at any time.  There are essentially two costs here: accessing the readonly static ```Log```, and calling ```IsEnabled()```, which just reads from an instance field (https://github.com/dotnet/coreclr/blob/43b39a73cbf832ec13ec29ed356cb75834e7a8d7/src/mscorlib/src/System/Diagnostics/Eventing/EventSource.cs#L274).  Given all that, the only caching we can do that can help in some situations is to cache the result of accessing Log into a local, for cases where multiple calls are made to Log in the same method.  But that doesn't happen with much frequency here, so I went for the simple approach.